pkgs/by-name/bu/buildstream/package.nix

```diff
@@ -41,6 +41,7 @@ python3Packages.buildPythonApplication rec {
     jinja2
     markupsafe
     packaging
+    pkgs.buildbox
```

Comment:
this is not overrideable, please put `[ buildbox ] ++` in front of `with python3Packages;`

Reply:
@pbsds I've fixed this in 0b67044, please re-review :-)